loops-dane-alex.js

```diff
@@ -0,0 +1,92 @@
+// Write a for loop that logs each number from 1 - 20.
+
+// for (let i = 1; i < 21; i++) {
+//     console.log(i)
+// }
+
+// Write a for loop that logs the result of each number from 1 - 20 tripled.
+
+// for (let i = 1; i < 21; i++) {
+//     console.log(i * 3)
+// }
+
+// Create a for loop that logs each even number from 1-20, and in the place of every odd number, returns the word "ODD" Expected output --> ODD, 2, ODD, 4, ODD, 6 ...etc
+
+// for (let i = 1; i < 21; i++) {
+//     if (i % 2 === 0) {
+//         console.log(i)
+//     } else {
+//         console.log("ODD")
+//     }
+// }
+
+
+// Looping over an array. Consider this variable:
+// var nums = [3, 57, -9, 20, 67]
+
+// Create a loop that will log the highest number from the array. Expected output --> 67
+var nums = [3, 57, -9, 20, 67]
+var largestnum
+for (let i = 0; i < nums.length; i++) {
+    if (!largestnum) { largestnum = nums[i] }
+    if (nums[i] > largestnum) { largestnum = nums[i] }
+}
+console.log(largestnum)
+
+
+// Create a loop that will log the lowest number from the array Expected output --> -9
+
+var nums = [3, 57, -9, 20, 67]
+var lowestnum
+for (let i = 0; i < nums.length; i++) {
+    if (!lowestnum) { lowestnum = nums[i] }
+    if (nums[i] < lowestnum) { lowestnum = nums[i] }
+}
+console.log(lowestnum)
```

Comment:
Awesome job understanding to place the console.log() outside the for loop to show one result vice all the iterations.